README.md

```diff
@@ -180,6 +182,52 @@ This is how the script knows we want to nest objects
 }
 ```
 
+##### Array
+
+Defines an Array of objects to be created with the same type.
+
+**options**
+- `size: Number` How many objects to create. **required**
+
+*Extending the company model a little further:*
+
+```json
+{
+  "company": {    
+    "type": "Object",
+    "value": {
+      "name": {    
+        "type": "faker",
+        "value": "company.companyName"
+      },
+      "addresses": {
+        "type": "Array",
+        "options": {"size": 10},
```

Comment:
I know .json format is sick, we should definitely modify it to support .js, but please break "size" declaration to a new line, my eyes hurt, ha ha.